apps/omg_api/config/test.exs

```diff
@@ -3,5 +3,7 @@ use Mix.Config
 config :omg_api,
   deposit_finality_margin: 1,
   exiters_finality_margin: 2,
-  ethereum_status_check_interval_ms: 50,
+  ethereum_events_check_interval_ms: 50,
```

Comment:
What's the reason we keep those intervals in 2:1 length ratio?

Reply:
hm, there isn't a hard requirement of the 2:1. It is rather a coincidence.

It sort of replicates the `block_queue_eth_height_check_interval_ms` and `coordinator_eth_height_check_interval_ms` being done "less frequently", because they need to pick up an advance of Ethereum, rather than some internal workings of our syncing like `ethereum_events_check_interval_ms` does